packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -380,6 +381,14 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     );
   }
 
+  // Render Chevron Down Icon
+  @autobind
+  private _onRenderChevronDownIcon(props: IDropdownProps): JSX.Element {
```

Comment:
Would it be a good idea to remove the props parameter here if it's not used?

Reply:
as long as you'll still able to send the props to the props.onRender.